### ISSUE TITLE
ancillary data

### ISSUE THREAD

**Issue by hoseinsalehi** [KMarkert/servir-vic-training#12]
Hello @kel markert
I have questions about ancillary data.
  You have said in the training files that we will provide data on the leaf area index and the Albedo  from the modis and to be used monthly in a tif file.()
My question is that the data is available every 8 days or 16 days, what can I do to make it monthly?
Do tif format data include time series?
I have problems here, are you guiding me?

**Comment by KMarkert** [comment]
@hoseinsalehi the provided data already should be monthly values. So, with LAI for example, there should be [12 files](https://github.com/KMarkert/servir-vic-training/tree/master/data/ancillary/LAI) named `Nyando_LAI_MM.tif` where MM is the month.

When using the `format_veg_params` and `make_veg_lib` functions you should just have to point it to the albedo and LAI folders with the 12 tif files that correspond to the months.

**Comment by hoseinsalehi** [comment]
I created 12 TIF files for LAI and Albedo and placed them in their folders.
I encountered the following error after entering the command in the terminal.
What changes should I apply to my data?
Thank you.
![vegliberror](https://user-images.githubusercontent.com/38522226/57967780-0517e800-7978-11e9-95d4-6dde213989c4.png)

**Comment by hoseinsalehi** [comment]
For 12 months, I downloaded LAI and Albedo in the tif format of Modis, and in separate folders like you, to generate
 veg.lib file.
But I encountered an error.
I have uploaded your input files as well as screenshots of errors.
What steps should I take to resolve the problem?
[Ancillary.zip](https://github.com/KMarkert/servir-vic-training/files/3194037/Ancillary.zip)
![Screenshot from 2019-05-18 15-17-12](https://user-images.githubusercontent.com/38522226/57969409-d146bd80-798b-11e9-941c-44ee0c9168a0.png)
